src/systests/test_client_userbehavior.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2016, 2017, 2018, 2019 Fraunhofer FKIE
```

Comment:
We already have a test_userbehavior.py. Maybe merge or rename?

Reply:
I do not exactly understand what the test_userbehavior.py is testing but I guess the two tests could be merged.